### ISSUE TITLE
Seafile-like user separated logins?

### ISSUE THREAD

**Issue by jult** [syncthing/syncthing#2970]
The reason seafile has an advantage over syncthing now is because it offers more users their own web-based or client-login to a server, and their files are per user based, separated, and they offer client-side encryption.

**Comment by calmh** [comment]
Ok? I'm not sure what the request is here. Please expand on it.

**Comment by jult** [comment]
Well, I've been a happy user of syncthing and its featureset, except for one thing;
I'm only one user, in a household of several potential other users that would all like to store their stuff on the same syncthing server (a NAS, actually). Who in their right mind would install a separate web-server for each user that wants to sync stuff? Seafile offers me a user/login, and then I see my synced folderset, which is only visible for me. Plus, it encrypts stuff before it gets stored on that server, which I would also prefer to have as an option in syncthing.

Basically, what it boils down to is that syncthing doesn't really seem to want to exist as a server-application, while everybody I personally know that uses syncthing, has one instance on a central server to sync to/from.

**Comment by canton7** [comment]
Is this a feature request? "This other application is better than you because ..." doesn't sound like "Please consider implementing ... because ..." to me :smile:

**Comment by jult** [comment]
To stir things up a bit. Seems to be working. The fact that you've looked there is already promising to me. But still, you're pretending you're dumb on this, while I know you're not. You *know* what people want, but both sides put their hands in their ears and say "nana-nanana, I don't hear you".

I've tried this before, back then I was polite. Doesn't work.

**Comment by canton7** [comment]
So you're deliberately being rude? Please don't.

**Comment by calmh** [comment]
> syncthing doesn't really seem to want to exist as a server-application

I think this is the correct interpretation. The server-to-many-clients use case is different, and in some respects easier. There are, as you've identified, probably better solutions out there for that space. Seafile might be one, Owncloud comes to mind as well.
